component/commons/handler/src/main/java/org/wso2/carbon/identity/password/expiry/validation/handler/PasswordExpiryPolicyUtils.java

```diff
@@ -0,0 +1,65 @@
+package org.wso2.carbon.identity.password.expiry.validation.handler;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+import org.wso2.carbon.identity.application.common.model.IdentityProviderProperty;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationManagementUtil;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManager;
+
+public class PasswordExpiryPolicyUtils {
+    private static final Log log = LogFactory.getLog(PasswordExpiryPolicyUtils.class);
+
+    private PasswordExpiryPolicyUtils(){}
+
+    public static String getIdentityEventProperty(String tenantDomain, String propertyName) {
+
+        // Retrieving properties set in identity event properties
+        String propertyValue = null;
+        try {
+            ModuleConfiguration moduleConfiguration = IdentityEventConfigBuilder.getInstance()
+                    .getModuleConfigurations(PasswordExpiryValidationConstants.PASSWORD_CHANGE_EVENT_HANDLER_NAME);
+
+            if (moduleConfiguration != null) {
+                propertyValue = moduleConfiguration.getModuleProperties().getProperty(propertyName);
+            }
+        } catch (IdentityEventException e) {
+            log.warn("An error occurred while retrieving module properties");
+            if (log.isDebugEnabled()) {
+                log.debug("An error occurred while retrieving module properties because " + e.getMessage(), e);
+            }
+        }
+        return propertyValue;
```

Comment:
I feel this method is generic and should already available in the product itself? can you check. if similar method is there we can reuse it

Reply:
Otherwise when we change the configuraion files in the product we have to attend multiple places.

Reply:
Seems like there is no such method

Reply:
The module configs should already be available in the super class I guess can you check it? This seems unnecessory.